src/calliope/model.py

```diff
@@ -372,9 +371,7 @@ def solve(self, force: bool = False, warmstart: bool = False, **kwargs) -> None:
 
         # Add additional post-processed result variables to results
         if results.attrs["termination_condition"] in ["optimal", "feasible"]:
-            results = postprocess_results.postprocess_model_results(
-                results, self.inputs, self.config.solve.zero_threshold
-            )
+            results = postprocess.postprocess_model_results(results, self)
```

Comment:
Wouldn't it make more sense to have the `if model.config.solve.postprocessing_active` check here, along with the check for optimal termination condition, rather than inside `postprocess.postprocess_model_results`?